ts/Accessibility/Accessibility.ts

```diff
@@ -479,7 +484,13 @@ namespace Accessibility {
         } else {
             // Just hide container
             this.renderTo.setAttribute('aria-hidden', true);
+
+            // If a11y has been enabled = true, and is now enabled = false
+            if (this.wasAccessibilityEnabled === false) {
```

Comment:
```suggestion
            if (!this.wasAccessibilityEnabled) {
```

Wouldn't if (!this.wasAccessibilityEnabled) be better than if (this.wasAccessibilityEnabled === false)?